src/data/services/lms/types/pageData.ts

```diff
@@ -88,10 +94,10 @@ export interface AssessmentsData {
       stepScore: { earned: number, possible: number },
       assessment: AssessmentData,
     },
-    peer?: {
+    peers?: {
       stepScore: { earned: number, possible: number },
       assessments: AssessmentData[],
-    },
+    }[],
     peerUnweighted?: {
       stepScore: null,
       assessmenst: AssessmentData[],
```

Comment:
I'm aware this was broken before, but:
```suggestion
      assessments: AssessmentData[],
```